cli/bpmetadata/types.go

```diff
@@ -68,11 +75,54 @@ type BlueprintDescription struct {
 	Architecture []string `json:"architecture,omitempty" yaml:"architecture,omitempty"`
 }
 
+type BlueprintDeploymentTime struct {
```

Comment:
Let's add some some unit of time as a suffix so it is clear if it is in seconds, minutes etc
I am thinking `BlueprintTimeEstimateSecs`?

Reply:
done